files/KoreBuild/scripts/common.sh

```diff
@@ -61,7 +61,8 @@ __ensure_macos_version() {
 
 __get_dotnet_sdk_version() {
     local src="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-    version=$(< "$src/../config/sdk.version" head -1 | sed -e 's/^[[:space:]]*//' -e 's/[[:space:]]*$//')
+
+    version=$(< "$src/../../../global.json" head -1 | grep -Po '"version":.*?[^\\]",' global.json)
```

Comment:
This only works when building inside aspnet/BuildTools. You'll need to look for the global.json file in the repo root, not a relative path to this script.

Also, it would be better to use `jq` (and fallback to `python`) to read the json file. See https://github.com/aspnet/BuildTools/blob/d9eb8a53eca84f61a7b0845c99fc43a3ec51e353/scripts/bootstrapper/run.sh#L219-L234 for example.